src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1723,11 +1785,21 @@
 helps['network vnet show'] = """
     type: command
     short-summary: Get the details of a virtual network.
+    long-summary: To learn more about virtual networks and settings, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-network#a-name--view-vnetaview-virtual-networks-and-settings
+    examples:
+        - name: Typical usage
+          text: >
+            az network vnet show -n myVNet -g MyResourceGroup
 """
 
 helps['network vnet update'] = """
     type: command
     short-summary: Update a virtual network.
+    long-summary: To learn more about managing virtual networks, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-network
+    examples:
```

Comment:
Remove or improve example. Suggestion: If there is a way to host DNS in Azure, maybe include a command to first extract available DNS servers for a region and then set those?

Reply:
We don't typically have a lot of "scenario examples" where we use multiple commands. There are a couple instances but they are kind of wonky. These are more for examples of using the commands. I would suggest:
1. Add multiple DNS servers (shows usage of space separated lists)
2. Add usage of the flags (ddos or vm protection)
3. Add a generic update usage (which can always use more examples).

Reply:
If this is the case, would it be useful to remove or streamline scenario-focused examples out of the CLI? There's currently some discussion ongoing in https://github.com/Azure/azure-docs-cli-python/pull/552 about the best way to onboard, and maybe moving some scenario-based examples into a tutorial, sample, or HOWTO would be appropriate.

Reply:
Left, for now.